griptape/artifacts/generic_artifact.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from typing import Any
+
+from attrs import define, field
+
+from griptape.artifacts import BaseArtifact
+
+
+@define
+class GenericArtifact(BaseArtifact):
+    value: Any = field(metadata={"serializable": True})
```

Comment:
What if its not serializable?

Reply:
I don't know off the top of my head but it'd be however Marshmallow handles it via `field.Raw`. We _do_ need it serializable though, so I think I'd rather just deal with it when we come across a value that is non-intuitively serializable.